crates/core/src/models/test_data/conversion/eth/transaction_receipt.json

```diff
@@ -0,0 +1,14 @@
+{
+  "transactionHash": "0x03204b4c0e379c3a5ccb80d08661d5a538e95e2960581c9faf7ebcf8ff5a7d3c",
+  "blockHash": "0x000000000000000000000000000000000000000000000000000000000000000d",
+  "blockNumber": "0x000000000000000000000000000000000000000000000000000000000000000d",
+  "from": "0x54b288676b749def5fc10eb17244fe2c87375de1",
+  "cumulativeGasUsed": "0xf4240",
+  "gasUsed": "0x7a120",
+  "contractAddress": "0x5fbdb2315678afecb367f032d93f642f64180aa3",
+  "logsBloom": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+  "logs": [],
```

Comment:
shouldn't logs have the events of the tx receipt?

Reply:
Not in this case, since the emitted events are pure Starknet events, they are not Eth events. The transaction receipt is based on transactions done to a [Counter contract](https://github.com/kkrt-labs/kakarot/blob/main/tests/integration/solidity_contracts/PlainOpcodes/Counter.sol), which doesn't emit any logs.